proxy/src/schema/registry.rs

```diff
@@ -158,18 +176,24 @@ impl Registry {
     }
 }
 
-#[test]
-fn test_register_project() {
+#[cfg(test)]
+mod tests {
+    use crate::schema::registry::Registry;
+    use radicle_registry_client::ed25519;
     use radicle_registry_client::Client;
 
-    // Test that project registration submits valid transactions and they succeed.
-    let client = Client::new_emulator();
-    let registry = Registry::new(client);
-    let alice = ed25519::Pair::from_legacy_string("//Alice", None);
-    let result = futures::executor::block_on(registry.register_project(
-        &alice,
-        "hello".into(),
-        "world".into(),
-    ));
-    assert!(result.is_ok());
+    #[test]
+    fn test_register_project() {
```

Comment:
Dunno if this is the right place for it, but we need to extend the tests to be more ellaborate and actually assert that we can get out what we put in and expect to get back from the Registry.

Reply:
Yes, fair point. I'll extend the tests. It makes sense to include that extension in the PR, to me, as it's testing features this PR adds.